test/unix/test_lwt_bytes.ml

```diff
@@ -0,0 +1,248 @@
+(* This file is part of Lwt, released under the MIT license. See LICENSE.md for
+   details, or visit https://github.com/ocsigen/lwt/blob/master/LICENSE.md. *)
+
+open Test
+
+let suite = suite "lwt_bytes" [
+    test "create" begin fun () ->
+      let len = 5 in
+      let buff = Lwt_bytes.create len in
+      let len' = Lwt_bytes.length buff in
```

Comment:
Just a note, it's a bit more "authoritative" to test this using [`Bigarray.Array.dim`](http://caml.inria.fr/pub/docs/manual-ocaml/libref/Bigarray.Array1.html#VALdim) instead of `Lwt_bytes.length`. Not suggesting to actually change the test case, not worth the time :)